packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -107,11 +107,11 @@ export const EventMeta = ({
     : isHalfFull
     ? "text-yellow-500"
     : "text-bookinghighlight";
-  const userLocale = session.data?.user.locale ?? navigator.language;
+  const userLocale = locale ?? navigator.language;
   const translatedDescription = (event?.fieldTranslations ?? []).find(
     (trans) =>
       trans.field === EventTypeAutoTranslatedField.DESCRIPTION &&
-      i18nLocales.includes(trans.targetLocale) &&
+      i18nLocales.locale.targets.includes(trans.targetLocale) &&
```

Comment:
1. i18nLocales name is misleading
2. `i18n.locale.targets.concat([i18n.locale.source])`